src/test/regress/sql/multi_create_table_shadow.sql

```diff
@@ -0,0 +1,290 @@
+--
```

Comment:
Why do we need to create this file?

Similar to comments for multi_test_helpers, we can use `\include` to avoid duplication of definitions.